### ISSUE TITLE
Small number of weird PAR time series

### ISSUE THREAD

**Issue by gregbritten** [brorfred/mapps_forest#2]
Hello. I am seeing some weird values in some PAR time series (example images pasted below). They all appear to be coming from  some Icelandic samples. 

![image](https://github.com/brorfred/mapps_forest/assets/19557522/df27fd13-bd33-4e68-828d-7b7a19e13bd3)

![image](https://github.com/brorfred/mapps_forest/assets/19557522/708ba5a7-491b-4529-80ac-05a05d52909e)

![image](https://github.com/brorfred/mapps_forest/assets/19557522/93e6af73-772f-453d-8e42-cb79f733a3e8)

**Comment by brorfred** [comment]
Hm, is this in the north hemisphere and is day Day of Year? I don't understand why PAR lower in the summer in that case.

**Comment by brorfred** [comment]
![rfig](https://github.com/brorfred/mapps_forest/assets/832870/eebc4b15-34cb-45e2-9864-aa3d3f4f4202)

This is what I get if I plot PAR vs Day of Year for data north of the equator. Colors represent latitude.

**Comment by gregbritten** [comment]
No, I am plotting the time series back in time from individual locations. So the x axis is days preceding, not Julian day. You can see an example of the whacky time series here: 
```
import netCDF4 as nc
import numpy as np
import matplotlib.pyplot as plt

f     = nc.Dataset('mapps_upstream_sat_365.nc')
par = f.variables['par']

plt.plot(par[647,:])
```
![image](https://github.com/brorfred/mapps_forest/assets/19557522/030ae7e7-4c32-4c0c-8cc9-023d2db167a4)

There aren't that many - seemingly all around Iceland.

**Comment by brorfred** [comment]
Hi Greg,

I've finally figured out what's going on with the anomalous data- there seems to be a bug in NASA's processing that creates edge effects. You can just filter out the data points, or I can try to remove them.
[edge_effect..pdf](https://github.com/brorfred/mapps_forest/files/12262521/edge_effect.pdf)

I've also add a jupyter notebook with the analysis:

https://github.com/brorfred/mapps_forest/blob/main/check_anamolous_par.ipynb

:-)B
